Source/Urho3D/Graphics/AnimationController.h

```diff
@@ -63,10 +64,22 @@ struct URHO3D_API AnimationParameters
     AnimationParameters& KeepOnZeroWeight();
     /// @}
 
-    /// Animation to be played.
-    /// Animation can be replicated over network only if is exists as named Resource on all machines.
-    Animation* animation_{};
-    StringHash animationName_;
+    /// Getters for read-only properties.
+    /// Use constructor to set Animation to the parameters.
+    /// @{
+    Animation* GetAnimation() const { return animation_; }
+    StringHash GetAnimationName() const { return animationName_; }
+    const WrappedScalar<float>& GetAnimationTime() const { return time_; }
+    /// @}
+
+
+    /// Time operations.
+    /// @{
+    float GetTime() const { return time_.Value(); }
```

Comment:
We expose min time and max time in Editor already. I guess we want to have API for this as well?

Reply:
You can read the time_ value via GetAnimationTime. I don't see any place where this gets written.